include/secp256k1_bulletproofs.h

```diff
@@ -93,14 +93,24 @@ SECP256K1_API size_t secp256k1_bulletproofs_rangeproof_uncompressed_proof_length
 ) SECP256K1_ARG_NONNULL(1);
 
 /** Produces an uncompressed rangeproof. Returns 1 on success, 0 on failure.
- *  Args:   ctx: pointer to a context object (cannot be NULL)
- *               gens: pointer to the generator set to use, which must have 2*n_bits generators (cannot be NULL)
- *    asset_gen: pointer to the asset generator for the CT commitment (cannot be NULL)
- *  Out:      proof: pointer to a byte array to output the proof into
- * In/OUt:  plen: pointer to the size of the above array; will be set to the actual size of
- *              the serialized proof. To learn this value in advance, to allocate a sufficient
- *              buffer, call `secp256k1_bulletproofs_rangeproof_uncompressed_proof_length` or
- *              use `SECP256K1_BULLETPROOFS_RANGEPROOF_UNCOMPRESSED_MAX_LENGTH`
+ *  Args:      ctx: pointer to a context object
+ *            gens: pointer to the generator set to use, which must have 2*n_bits generators
+ *       asset_gen: pointer to the asset generator for the Pedersen/CT commitment
+ *  Out:     proof: pointer to a byte array to output the proof into
+ *  In/Out:   plen: pointer to the size of the above array; will be set to the actual size of
+ *                  the serialized proof. To learn this value in advance, to allocate a sufficient
+ *                  buffer, call `secp256k1_bulletproofs_rangeproof_uncompressed_proof_length` or
+ *                  use `SECP256K1_BULLETPROOFS_RANGEPROOF_UNCOMPRESSED_MAX_LENGTH`
+ *  In:     n_bits: value committed in the Pedersen commitment
+ *           value: value committed in the Pedersen commitment
+ *       min_value: minimum value of the range being proven
+ *          commit: the Pedersen commitment being proven
+ *           blind: blinding factor for the Pedersen commitment
+ *           nonce: seed for the RNG used to generate random data during proving
+ *        enc_data: up to 32 bytes of extra data which may be decrypted by a verifier who knows `nonce`
```

Comment:
Does `enc_data` need to be exactly 32 bytes, since it'll be used by `secp256k1_scalar_set_b32` when non-null?

```suggestion
 *        enc_data: 32-byte array of extra data which may be decrypted by a verifier who knows `nonce`
```